src/main/java/com/metamx/collections/bitmap/WrappedRoaringBitmap.java

```diff
@@ -126,6 +143,9 @@ public int size()
   @Override
   public void serialize(ByteBuffer buffer)
   {
+    if (compressRunOnSerialization) {
+      bitmap.runOptimize();
```

Comment:
A false means that you called runOptimize but it did not change the bitmap, as per the JavaDoc documentation.

http://lemire.me/docs/RoaringBitmap/org/roaringbitmap/buffer/MutableRoaringBitmap.html#runOptimize--

The return value can be safely ignored.